components/controls/buttons.css

```diff
@@ -281,6 +281,10 @@
   color: #fff;
 }
 
+.button.button-icon.disabled {
+  background: transparent !important;
+}
```

Comment:
Looks good, but the icon becomes white on hover, which means invisible in many cases. I'd make the icon always grey, but maybe it's better to add the background back on hover?

Reply:
Oh, good catch. I think keeping it gray makes more sense. Why have a hover state if it's disabled in the first place? Hover state suggests interactivity.